subsystem/intake.py

```diff
@@ -0,0 +1,60 @@
+import config
+import constants
+from toolkit.subsystem import Subsystem
+from toolkit.motors.rev_motors import SparkMax, SparkMaxConfig
+
+INNER_CONFIG = SparkMaxConfig(.5, 0, 0)
+OUTER_CONFIG = SparkMaxConfig(.5, 0, 0)
+
+class Intake(Subsystem):
+    def __init__(self):
+        super().__init__()
+
+        self.inner_motor: SparkMax = SparkMax(
+            can_id=config.inner_intake_id,
+            config=INNER_CONFIG
+        )
+
+        self.outer_motor_front: SparkMax = SparkMax(
+            can_id=config.outer_intake_front_id,
+            config=OUTER_CONFIG
+        )
+
+        self.outer_motor_back: SparkMax = SparkMax(
+            can_id=config.outer_intake_back_id,
+            config=OUTER_CONFIG
+        )
+    
+    def init(self):
```

Comment:
Cool. As we discussed today, make sure to add the beam-break sensor. This would be an input device from the DIO port on the RoboRIO. Use `from wpilib import DigitalInput` as that connection. Also, make sure to write methods to check when it is true or false.